eve/main.yml

```
@@ -671,7 +671,9 @@ models:
       command: |
         ssh -F ssh_config bootstrap <<ENDSSH
         sudo kubectl --kubeconfig /etc/kubernetes/admin.conf \
-        patch node "$NODE_NAME" --patch '{"spec": {"taints": []}}'
+        patch node "$NODE_NAME" --patch '{"spec": {"taints": []}}' &&
+        sudo kubectl --kubeconfig /etc/kubernetes/admin.conf \
+        label node --overwrite  "$NODE_NAME" "node-role.kubernetes.io/node="
```

Why not do this within the patch itself?

```suggestion
        patch node "$NODE_NAME" --patch '{
          "metadata": {"labels": {"node-role.kubernetes.io/node": ""}},
          "spec": {"taints": []}
        }'
```

---

Right, I could have done this in a single patch, does not really matter :smile: